app/src/main/java/org/kiwix/kiwixmobile/search/AutoCompleteAdapter.java

```diff
@@ -82,9 +83,17 @@ public String getItem(int index) {
     }
   }
 
-  @Override
-  public Filter getFilter() {
-    return mFilter;
+  @Override public int getItemCount() {
+    return mData.size();
+  }
+
+  public class ViewHolder extends RecyclerView.ViewHolder {
+    TextView txt;
```

Comment:
Use a better name. It does not tell me what is it.

Reply:
a better name for txt? text would work?

Reply:
`title` would be good.